### ISSUE TITLE
Warning: foreach() argument must be of type array|object, null given

### ISSUE THREAD

**Issue by laryn** [backdrop-contrib/print#30]
After a fresh install, when configuring PDF settings:

> Warning: foreach() argument must be of type array|object, null given in form_type_checkboxes_value() (line 2448 of /var/www/html/core/includes/form.inc).

**Comment by argiepiano** [comment]
Yeah, I noticed this one when I was working on #13. But this was there before merging #13.
